### ISSUE TITLE
GUI text is too small/large when the window is resized at run-time

### ISSUE THREAD

**Issue by Calinou** [stuntrally/stuntrally#54]
Unlike #53, this is not a new issue – it also occurred in 2.6 and prior versions.

Resizing the window will not adjust text size accordingly, so some text will appear to be too small or large. Example when going from 2560×1440 to 1024×768:

![image](https://user-images.githubusercontent.com/180032/170837553-2f5ab70c-0e8e-4c62-b0eb-525d827dcead.png)

Like #53, we can work around this by disallowing window resizing.

**Comment by cryham** [comment]
Yes I know. I hate all those issues because of all those tricks we coded just to have MyGui resizing GUI and fonts (as it should in the first place).
